pkg/web/static/js/constants.js

```diff
@@ -327,19 +327,24 @@ export const naturalPersonNameTypeArray = Object.entries(IDENTIFIER_TYPE)
 
 // Reject error codes
 export const REJECT_CODES = {
-  REJECTED: "Rejected",
-  UNKNOWN_WALLET_ADDRESS: "Unknown Wallet Address",
-  UNKNOWN_IDENTITY: "Unknown Identity",
-  UNKNOWN_ORIGINATOR: "Unknown Originator",
-  UNKNOWN_BENEFICIARY: "Unknown Beneficiary",
-  UNSUPPORTED_CURRENCY: "Unsupported Currency",
-  EXCEEDED_TRADING_VOLUME: "Exceeded Trading Volume",
-  COMPLIANCE_CHECK_FAIL: "Compliance Check Fail",
-  NO_COMPLIANCE: "No Compliance",
-  HIGH_RISK: "High Risk",
-  OUT_OF_NETWORK: "Out of Network",
+  REJECTED: "Default rejection - no specified reason for rejecting the transaction",
```

Comment:
This was updated to be more in line with what's listed in the API docs. The values would be displayed in the error envelope or at the top of the repair form.